Tasks/BMICalculator.js

```diff
@@ -1,10 +1,9 @@
-const weight = 50;
-const height = 3;
-
-function BMI(weight, height) {
-  console.log((weight / (height * height)).toFixed(3));
+function getBMI(weight, height) {
+  if (height <= 0) return "height should be greater than zero." 
```

Comment:
this function has side effects it is returning two different types it should return only value of BMI. If height and weight is not the expected input, handle through throw keyword